app/assets/v2/js/grants/matching_funds.js

```diff
@@ -273,7 +273,7 @@ Vue.mixin({
 });
 
 if (document.getElementById('gc-matching-funds')) {
-  props: ['grant'],
```

Comment:
do we need need this ?

Reply:
@thelostone-mc Have just discuss this with @gravityblast , it seems to have been added there by mistake.
It actually does not do anything.